lib/bitters/generator.rb

```diff
@@ -10,13 +10,14 @@ class Generator < Thor
     map ['delete'] => :remove
 
     desc 'install', 'Install Bitters into your project'
-    method_options path: :string
+    method_options path: :string, syntax: :string
     def install
       if bitters_files_already_exist?
         puts "Bitters files already installed, doing nothing."
       else
         install_files
-        puts "Bitters files installed to #{install_path}/base"
+        puts "Bitters files installed to #{install_path}/base."
+        puts "Converted files to Sass syntax." if options[:syntax].downcase == 'sass'
```

Comment:
I'll update my request to fix the violations in the style guide. Stay tuned.
On Mon, Jan 12, 2015 at 1:25 PM Joshua Ogle <notifications@github.com>
wrote:

> In lib/bitters/generator.rb
> <https://github.com/thoughtbot/bitters/pull/138#discussion-diff-22809701>:
>
> >      def install
> >        if bitters_files_already_exist?
> >          puts "Bitters files already installed, doing nothing."
> >        else
> >          install_files
> > -        puts "Bitters files installed to #{install_path}/base"
> > +        puts "Bitters files installed to #{install_path}/base."
> > +        puts "Converted files to Sass syntax." if options[:syntax].downcase == 'sass'
>
> The thoughtbot styleguide also suggests not using conditional modifiers
> (the conditional at the end of the line), so this might be better as:
>
> if options[:syntax].downcase == "sass"
>   puts "Converted files to Sass syntax."
> end
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thoughtbot/bitters/pull/138/files#r22809701>.
>